2022/incoming/at-the-present-factory.pod

```diff
@@ -0,0 +1,127 @@
+Author: lichtkind@cpan.org
+Title: At the Present Factory
+Topic: charts, automation
+
+=encoding utf8
+
+=head1 At the Present Factory
+
+Father Christmas is very old-school and so is his Perl code. There are no
+towering software engineering marvels - just a bunch of wonderful scripts:
+small, meticulously written, very readable and changeable.
+And because there are so well organized, his system scales up to billions 
+cheering children.
+
+Reporting about such high volume of numbers has to be graphical - charts are needed.
+They are getting mass produced by scripts that convert one dataset into one chart
+of a particular layout and design by a call like C<perl script.pl data.tsv > chart.png>.
```

Comment:
Something is wrong with this code snippet

Reply:
as in my latest comment changed it with &gt; whouly be good now